models/database.go

```diff
@@ -111,6 +111,8 @@ var databaseSchema = [][]string{
 			username VARCHAR CHECK (username <> ''),
 			password VARCHAR CHECK (password <> ''),
 			metrics_url VARCHAR CHECK (metrics_url <> ''),
+			tls BOOLEAN,
```

Comment:
let's create new migration instead of changing existed one.

Reply:
@percona-csalguero To do that, add a new slice element below. Also, make sure it is compatible with existing data. Something like this should work:
* add columns with DEFAULT false;
* remove default values.